### PR TITLE
hhdb update

### PR DESCRIPTION
**Pull request by mayoub**
update xsec

### CODE REVIEW

**Comment by mayoub** [comment]
update hhdb

File: hhdb/xsec/13TeV_v2_AOD.txt [hunk 1]
```diff
@@ -0,0 +1,82 @@
+301156 	50000 	1.4307e-06*1.0*1.0 	34947927.59 	mc15_13TeV.301156.PowhegPythia8EvtGen_AZNLOCTEQ6L1_Wplustaunu_4000M4500.merge.AOD.e3663_s2608_s2183_r6630_r6264
```
Comment (qbuat) [inline hhdb/xsec/13TeV_v2_AOD.txt]: We don't need this file.

File: hhdb/xsec/dataset_13TeV_v2 [hunk 2]
```diff
@@ -0,0 +1,59 @@
+mc15_13TeV.301074.PowhegPythia8EvtGen_AZNLOCTEQ6L1_Wplusenu_3000M3500.merge.DAOD_HIGG4D2.e3663_s2608_s2183_r6630_r6264_p2411
```
Comment (qbuat) [inline hhdb/xsec/dataset_13TeV_v2]: We don't need this file either.

Comment (qbuat) [inline hhdb/xsec/dataset_13TeV_v2]: Remove those two files and then I will merge.
Thanks!

**Comment by qbuat** [comment]
:+1:

**Pull request merged**